packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -19,6 +19,8 @@ For Log Stream Schemas, Okta uses [JSON Schema Draft 2020-12](https://json-schem
 
 Explore the Schemas API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/c85985861f9b277913ae)
 
+<ApiAuthMethodWarning />
```

Comment:
I noticed this notice is in a different location from the other notices. This is to avoid conflict with a notice that's already on the page, correct? (Just wanted to confirm!)

Reply:
Yes, I placed the auth note where I think is appropriate, but feel free to suggest another location that is more appropriate on the page.
Since this is a note regarding the curl examples, I usually placed it after the API overview/description to not confuse users and take focus away from the main objective of the API page. In some cases, there are several notes relating to functionality, so I placed the auth note after the Get Started section, where we provide a Postman collection with SSWS examples.

Reply:
Sounds good! Agree it's good where it is.